src/python/WMCore/BossAir/Plugins/SimpleCondorPlugin.py

```diff
@@ -100,10 +100,16 @@ def __init__(self, config):
 
         self.packageDir = None
 
-        if os.path.exists(os.path.join(getWMBASE(),
-                                       'src/python/WMCore/WMRuntime/Unpacker.py')):
-            self.unpacker = os.path.join(getWMBASE(),
-                                         'src/python/WMCore/WMRuntime/Unpacker.py')
+        # if agent is running in a container, Unpacker.py must come from a directory
+        # on the host so the condor schedd can see it
+        # config.General.workDir should always be bind mounted to the container
+        if config.Agent.isDocker:
+            unpackerPath = os.path.join(config.General.workDir + "/Docker/WMRuntime/Unpacker.py")
+        else:
```

Comment:
Eric, would you know what `getWMBASE()` is when running from a container?

Reply:
Oh, and did you test both of these cases?

Reply:
I tested both the Docker and "standard" VM deployment paths